docs/examples/tf/tf_distributed/mnist_example.py

```diff
@@ -0,0 +1,52 @@
+# all taken from https://www.tensorflow.org/guide/keras/functional
+import click
+
+
+@click.command()
+def train_dense_model_click():
+    return train_dense_model(batch_size=64)
+
+
+def train_dense_model(batch_size):
+    # limit imports oustide the call to the function, in order to launch quickly
+    # when using dask
+    import tensorflow as tf
+    from tensorflow import keras
+    from tensorflow.keras import layers
+    # model building
+    tf.keras.backend.clear_session()  # For easy reset of notebook state.
+
+    slurm_resolver = tf.distribute.cluster_resolver.SlurmClusterResolver(port_base=15000)
+    mirrored_strategy = tf.distribute.MultiWorkerMirroredStrategy(cluster_resolver=slurm_resolver)
```

Comment:
As mentioned @mypey on gitter we should employ `communication=tf.distribute.experimental.CollectiveCommunication.NCCL` here

Reply:
yes that's totally true, I didn't think of changing it in the PR before it was merged. We can have a small PR to fix this.

Reply:
#59